fiftyone/core/dataset.py

```diff
@@ -2999,11 +2999,12 @@ def delete_labels(
             ops = []
             if issubclass(label_type, fol._LABEL_LIST_FIELDS):
                 array_field = field + "." + label_type._LABEL_LIST_FIELD
+                query = {array_field: {"$exists": True}}
```

Comment:
not sure if this is applicable without looking at the rest of the aggregation/knowing the root, but shouldn't the schema be the same? (field should exist but may be null).

Reply:
Not sure what you're asking. These updates are looking for documents in a nested list field and deleting them by ID. So there needs to be a `query` to skip documents that are missing the top-level field, or else an error will occur.